_posts/blog/2021-04-21-jshell-accumulo-feature.md

```diff
@@ -0,0 +1,141 @@
+---
+Title: JShell Accumulo Feature
+Author: R. Dane Magbuhos
+Reviewers: Christopher Tubbs, Keith Turner
+---
+
+## Overview
+First introduced in Java 9+, [JShell][JShell Doc] is an interactive Read-Evaluate-Print-Loop (REPL) 
+Java tool that assess user's inputed declarations, statements, and expressions and outputs 
+the results. This tool provides a convenient way to test out and execute quick tasks with Accumulo
+in the terminal.
+
+## Major Features
+* During Accumulo build, produces a default JShell script called `jshell-init.jsh` 
+containing up-to-date [Accumulo Java APIs][public APIs] and [AccumuloClient][client] 
+build implementation
```

Comment:
Instead of mentioning how it is built, which isn't quite relevant to end users, it should instead mention that a default `conf/jshell-init.jsh` file is provided in our binary distribution tarball, and includes some initial imports for interacting with Accumulo's API.

It might be worth mentioning that JShell could be run manually using the user's installed `jshell` command from their installation of Java, but that by running it with `bin/accumulo jshell`, it will automatically pick up the `CLASSPATH` and other environment from the user's `conf/accumulo-env.sh` and automatically initialize with the provided `conf/jshell-init.jsh` file, which can also be customized by users.

Since all of this is something a user could already do themselves, I think the main focus of this blog post should be on the convenience that it offers to users, so that they don't have to do it all manually.